docs/layouts.rst

```diff
@@ -149,10 +151,10 @@ As you might expect, ``extends`` can be composed multiple times::
     x: 100
   plato:
     extends: socrates
-    x: += 10    # evaluates to 150
+    x: += 10    # evaluates to 110
   aristotle:
     extends: plato
-    x: += 20    # evaluates to 150
+    x: "*= 2"     # evaluates to 220, note that YAML requires quotes here
```

Comment:
copy/paste mistake in original docs? replaced with *= example instead of fixing.